generate-csv.js

```diff
@@ -58,6 +60,23 @@ function generateCSV() {
     })
 }
 
+function insertRow(rows, row) {
+    let newRow = {};
+    newRow.receiver = row.receiver
+    newRow.amount = row.amount
+    newRow.name = ''
+    newRow.token_type = 'erc20'
+    newRow.token_address = '0x333A4823466879eeF910A04D473505da62142069'
```

Comment:
Since you added this refactoring, I guess you can delete the same two rows from `writeToGnosisCSV()`?

```javascript
function writeToGnosisCSV(filePathGnosis, csvRows) {
    console.log('writeToGnosisCSV')

    // Add missing columns
    csvRows.forEach(function(row, index) {
        row.token_type = 'erc20'
        row.token_address = '0x333A4823466879eeF910A04D473505da62142069'
    })
    
    .
    .
    .
```

https://github.com/nation3/nationcred-instance/blob/8faa6c758f8750d33d581077c75aa1d4f0b322b4/generate-csv.js#L111

Reply:
Argh, we should have just deleted the other PR, that was deleted, VSC automerge put it back in. Done :D The code in the other PR was wrong, added the token address twice.